modules/setting/setting.go

```diff
@@ -767,7 +767,55 @@ func NewContext() {
 	MinPasswordLength = sec.Key("MIN_PASSWORD_LENGTH").MustInt(6)
 	ImportLocalPaths = sec.Key("IMPORT_LOCAL_PATHS").MustBool(false)
 	DisableGitHooks = sec.Key("DISABLE_GIT_HOOKS").MustBool(false)
+
+	InternalToken = sec.Key("INTERNAL_TOKEN").String()
```

Comment:
Why the codes are here?

Reply:
I did not understand.  What do you mean?

Reply:
This block is defined elsewhere in settings, I think it could be a copy/paste error. Not sure is this block of code is needed.